.npmrc

```diff
@@ -1,2 +0,0 @@
-audit-level = high
```

Comment:
Why are we removing the .npmrc config file? I think it is best to use that for our project config.
The pipeline will use that to run its scripts too. Maybe we can reuse the one we have in the update store service?

https://github.com/dvsa/cvs-svc-enquiry/blob/feature/CVSB-19222/.npmrc

Reply:
Addressed https://github.com/dvsa/cvs-tsk-update-store/pull/1/commits/c9ba1e53216fa6f1d412e93a875ecbb42caef4c2